ee/clickhouse/queries/paths/path_event_query.py

```diff
@@ -45,11 +45,6 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
             funnel_paths_timestamp,
         ]
         _fields += [f"{self.EVENT_TABLE_ALIAS}.{field} AS {field}" for field in self._extra_fields]
-        _fields += [
-            get_property_string_expr("events", field, f"'{field}'", "properties", table_alias=self.EVENT_TABLE_ALIAS)[0]
-            + f" as {field}"
-            for field in self._extra_event_properties
```

Comment:
Also needed to clean this up due to a potential SQL injection risk/usage of $ in column names.